### PR TITLE
Remove offline nodes from replica info after LoadBalanceTask done

### PR DESCRIPTION
**Pull request by yah01**
Signed-off-by: yah01 <yang.cen@zilliz.com>
#16637

### CODE REVIEW

**Comment by yah01** [comment]
rerun ut

**Comment by yah01** [comment]
rerun ut

File: internal/querycoord/task.go [hunk 1]
```diff
@@ -2208,13 +2208,57 @@ func (lbt *loadBalanceTask) postExecute(context.Context) error {
...
+			offlineNodes[nodeID] = struct{}{}
+		}
+		replicas := make(map[UniqueID]*milvuspb.ReplicaInfo)
+
 		for _, id := range lbt.SourceNodeIDs {
 			err := lbt.cluster.removeNodeInfo(id)
 			if err != nil {
 				//TODO:: clear node info after removeNodeInfo failed
-				log.Error("loadBalanceTask: occur error when removing node info from cluster", zap.Int64("nodeID", id))
+				log.Warn("loadBalanceTask: occur error when removing node info from cluster",
```
Comment (congqixia) [inline internal/querycoord/task.go:2221]: NIT: add err log field here.

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**